.env.example

```
@@ -1,2 +1 @@
-CF_EMAIL=your_cloudflare_account_email          # The email you use to login to Cloudflare
-CF_API_TOKEN=your_cloudflare_global_api_key     # Located at the bottom of https://dash.cloudflare.com/profile/api-tokens
+CF_API_TOKEN=your_cloudflare_api_token     # A scoped API token — needs User Details:Read and User API Tokens:Edit permissions
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)